### PR TITLE
 Fixed garbled characters in NON ASCII Code

### PR DESCRIPTION
**Pull request by yshingai**
Use jackson for JSON encoding

### CODE REVIEW

**Comment by chiragnaik** [comment]
@yshingai Please fix the build issue

**Review by chiragnaik — APPROVED** [review]

File: src/main/java/io/cnaik/service/CommonUtil.java [hunk 1]
```diff
@@ -66,7 +68,24 @@ public void send() {
...
+                {
+                    setText(responseMessageUtil.createTextMessage());
+                }
+            };
+            ObjectMapper mapper = new ObjectMapper();
+            mapper.configure(JsonGenerator.Feature.ESCAPE_NON_ASCII, true);
+            try {
+            json = mapper.writeValueAsString(response);
+            } catch (Exception e) {
+            }
```
Comment (Ryudo302) [inline src/main/java/io/cnaik/service/CommonUtil.java:88]: @yshingai Please add exception handling here (maybe just log?)

File: src/main/java/io/cnaik/service/CommonUtil.java [hunk 2]
```diff
@@ -66,7 +68,24 @@ public void send() {
...
+            Response response = new Response() {
+                {
+                    setText(responseMessageUtil.createTextMessage());
+                }
+            };
+            ObjectMapper mapper = new ObjectMapper();
+            mapper.configure(JsonGenerator.Feature.ESCAPE_NON_ASCII, true);
+            try {
+            json = mapper.writeValueAsString(response);
+            } catch (Exception e) {
```
Comment (Ryudo302) [inline src/main/java/io/cnaik/service/CommonUtil.java:87]: Please add exception handling here (maybe just log?)

File: src/main/java/io/cnaik/service/CommonUtil.java [hunk 3]
```diff
@@ -66,7 +68,24 @@ public void send() {
         if(false) {
             json = responseMessageUtil.createCardMessage();
         } else {
-            json = "{ \"text\": \"" + responseMessageUtil.createTextMessage() + "\"}";
+            class Response {
```
Comment (Ryudo302) [inline src/main/java/io/cnaik/service/CommonUtil.java:71]: @yshingai I suggest you to create a new file for this class. Probably in the package `io.cnaik.model.google`.

**Review by Ryudo302 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**